### DISCUSSION TITLE
NavigationItem disable

### DISCUSSION THREAD

**Discussion by mstfkhazaal** [filamentphp/filament#13318] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2.92

### How can we help you?

How can disabel NavigationItem?

**Comment by hotsaucejake** [comment, marked as answer]
https://filamentphp.com/docs/3.x/panels/navigation#disabling-resource-or-page-navigation-items

or

https://filamentphp.com/docs/3.x/panels/navigation#conditionally-hiding-navigation-items
